hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -55,33 +71,34 @@ def get(cls):
         return cls(
             s3_bucket=os.environ["THREAT_EXCHANGE_DATA_BUCKET_NAME"],
             s3_te_data_folder=os.environ["THREAT_EXCHANGE_DATA_FOLDER"],
-            collab_config_table=os.environ["THREAT_EXCHANGE_CONFIG_DYNAMODB"],
+            collab_config_table=os.environ["HMA_CONFIG_TABLE"],
         )
 
 
-def lambda_handler(event, context):
-    config = FetcherConfig.get()
+@dataclass
+class ThreatExchangeConfig(HMAConfig):
```

Comment:
Is `ThreatExchangeConfig` going to be used anywhere else?

Perhaps it is better located in a non-lambda-specific module?

Reply:
Originally had it as its own module, but then ran into some issue (couldn't remember) that brought me back here. My plan is to refactor this whole module to slim it down in a followup